fix #error gradient on Android

seanmonstar opened the pull request:
it's gradient doesn't go all the way down either. We must keep
the backgrounds on #wait, #delay, and #error, since they are
needed to cover the #formWrap background.

the same background of #error is added to #content, so that
even though #error doesn't extend all the way down, the gradient
still does, and it looks correct on Android again.

fixes #1721

/cc @shane-tomlinson

seanmonstar committed:
add background to #content div

it's gradient doesn't go all the way down either. We must keep
the backgrounds on #wait, #delay, and #error, since they are
needed to cover the #formWrap background.

the same background of #error is added to #content, so that
even though #error doesn't extend all the way down, the gradient
still does, and it looks correct on Android again.

fixes 1721

shane-tomlinson commented on resources/static/css/common.css:
```diff
@@ -357,7 +357,7 @@ footer .help {
-#wait, #delay, #error {
+#content, #wait, #delay, #error {
   background-color: #dadee1;
   background-image: url("/i/grain.png"), -moz-linear-gradient(top, #dadee1, #c7ccd0);
```
It looks like we need some gradients for webkit, opera, and ms here too.  How'd I miss that?

seanmonstar replied:
Oh yea, I didn't notice that either. Does the gradient override the grain.png? Or is the png transparent?

shane-tomlinson commented:
@seanmonstar - I'm testing this now, will have more to say.  I think the solution may be even simpler yet, but I want to check.

seanmonstar commented:
@shane-tomlinson curious what you have found.

Commit:
Make the error messages look right on all mobile browsers by removing the set height of the error message and its contents.

To the #wait, #error, and #delay screens, add the gradients necessary for non-Mozilla browsers.

issue #1721

seanmonstar committed:
Merge pull request #1 from shane-tomlinson/issue_1721_android_gradient

Issue 1721 android gradient

seanmonstar commented:
Yea, that looks right as well.

shane-tomlinson commented:
Re-opened as PR-1842.  Closing this one.
